samples/SocketsSample/Hubs/Streaming.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Channels;
+using Microsoft.AspNetCore.SignalR;
+
+namespace SocketsSample.Hubs
+{
+    public class Streaming : Hub
+    {
+        [return: Streaming]
```

Comment:
But if you think it's clear that `IAsyncEnumerable<T>` should be streamed, we can get rid of the attribute.